solutions/python-igraph/clique_percolation/clique_percolation.py

```diff
@@ -0,0 +1,64 @@
+import igraph as ig
+import itertools
+import matplotlib.pyplot as plt
+
+EXAMPLE_K = 3
+
+def k_communities(g, k, min_common_vertices):
+    """
+    Given a graph and size k, return a list of communities found through clique
+    percolation with at least min_common_vertices shared.
+    A community is the connected component of cliques where two cliques overlap
+    in at least k-1 vertices.
+    Each community is a set of vertices in that community
```

Comment:
I think lines 13-16 can be removed, because they don't add much in terms of explanation of the function.